src/Orleans/Statistics/RuntimeStatisticsGroup.cs

```diff
@@ -254,7 +254,7 @@ public void Dispose()
             numberOfInducedGCsPF.Dispose();
             largeObjectHeapSizePF.Dispose();
             promotedFinalizationMemoryFromGen0PF.Dispose();
-            this.cpuUsageTimer?.Dispose();
+            cpuUsageTimer?.Dispose();
```

Comment:
my bad... if we don't use the safe navigation operator then the warning will go away - the analyzer is dated.